CIP-0027.md

```diff
@@ -0,0 +1,62 @@
+
+A community standard for royalties&#39; functionality, that does not require smart contracts to implement.
+
+## Abstract
+
+This proposed standard will allow for uniform royalties&#39; distributions across the secondary market space. It is easy to implement using metadata only, and does not require a smart contract.  However, it is scalable to allow for the usage of a downstream smart contract, as needed by the asset creator. It has been developed with input from Artano, BuffyBot, CNFT.io, Digital Syndicate, Fencemaker, MADinArt, NFT-Maker.io, Hydrun, Tokhun, and many more.
+
+## Motivation
+
+There is a significant interest within the Cardano Community for an implementation of royalties distribution when a Cardano Asset is resold on the secondary market. It has become a common theme to see and hear statements that the only thing stopping artists from adopting Cardano, is that they are waiting for an implementation of royalties.  At the present time, smart contracts do not create a simple mechanism to implement royalties.  By developing a community standard, we can resolve the immediate need for royalties, and create a path forward for a potential future iteration of smart contracts.
+
+## Specification
+
+A new tag of 777 is proposed for this implementation.  The community guidelines have been agreed as follows:
+1) The royalties tags are to be written to an unnamed token, using the policy to be used for the intended Cardano Assets.
+2) Only the first minted set of instructions will be honored.  Any future updates or rewrites will be ignored.  This prevents a Cardano Asset maker from changing the royalties at a future date.
+3) Within this created asset will be the metadata for royalties distributions.  It will use a tag of 777, and then have two tags to identify the percentage of future sales requested as a royalty, and the payment address to forward those royalties to.  Those tags will be "pct" and "addr" respectively.
+4) The "pct" key tag can be any floating point value from 0.0 to 1.0, to represent between 0 and 100 percent.  For example, a 12.5 percent royalty would be represented with "pct": "0.125".
+5) The "addr" key tag can be a string value, or an array.  It is to include a single payment address.  By allowing for an array, the payment address can exceed the per line 64 character limitation.  This payment address could be part of a smart contract, which should allow for greater flexibility of royalties distributions, controlled by the asset creator.
+
+## Example JSON with string
+
+{
+	"777": {
+		"pct": "0.2",
+		"addr": "addr1v9nevxg9wunfck0gt7hpxuy0elnqygglme3u6l3nn5q5gnq5dc9un"
+	}
+}
+
+## Example JSON with array
+
+{
+	"777": {
+		"pct": "0.2",
+		"addr": [
+			"addr1q8g3dv6ptkgsafh7k5muggrvfde2szzmc2mqkcxpxn7c63l9znc9e3xa82h",
+			"pf39scc37tcu9ggy0l89gy2f9r2lf7husfvu8wh"
+		]
+	}
+}
+
+## Process Flow
+1) Create policy for planned assets.
+2) Mint no name token with community standard royalties metadata.
```

Comment:
Is there a reason that we cannot just add this "777" tag to the actual minting transaction of the token itself instead of minting and burning an empty token with the same policy first?

Reply:
You can.  But there is two significant reasons it has been proposed this way.  The first is chain bloat.  It would just be repeated data for however many NFT's are minted under the same policy.  Second, and probably more important, by having the creator do it at the outset, and only respecting the very first mint, the royalties become a known quantity.  It prevents a bad actor from rewriting their royalties across various assets.  It provides security for the community.  The best practice would be for the royalty token to have been minted and articulated to the community, prior to the sales of a given asset.

Reply:
Interesting. I can understand it being an issue for multiple tokens minted under the same policy. From my understanding, a bad actor shouldn't be able to rewrite the metadata anyway since platforms should only consider the minting transaction's metadata as legitimate, not any trading transactions. I see what you mean regarding the pre-minting of the royalty token though, I just wanted to understand why the extra complexity was needed. Thank you.

Reply:
Ah, I see the piece you're missing now.  You're correct that a transactions (send/receive) metadata does not change an NFT.  However, it is possible to rewrite the metadata on an NFT by reminting with the same policyID.name.  Two examples of this are Space Coins and the Chadi Nassar Surprise Self Portrait Drop Celebration.

In the case of Space Coins, it did not have any graphics attached to it initially.  But later on, more were minted (granted this one is a fungible), and the image info was added.  At that point, anyone that held a spacecoin had an updated copy with graphics.  In the second case, the original NFT was minted with a graphical image of a question mark.  The NFT's were sold and sent out.  Then when the time came, another token was minted with the same policyID.name with the now updated graphics.  The second asset was immediately deminted, leaving a quantity of one.  Same as the spacecoins, the latest mint became the relevant metadata.